### PR TITLE
Lint JS for putting exports last

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

adds a lint rule to require that module exports come last in the file.
ignore's flow type exports, since they need to be defined before they can be used.

## What is the value of this and can you measure success?

make exports easy to find

### CODE REVIEW

File: dev/eslint-plugin-guardian-frontend/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-plugin-guardian-frontend",
   "main": "index.js",
-  "version": "2.0.0",
+  "version": "3.1.0",
   "scripts": {
     "test": "jest"
```
Comment (SiAdcock) [inline dev/eslint-plugin-guardian-frontend/package.json]: Unrelated, do you think we should add Jest as a `devDependency` here? This script will only work if Jest is installed globally

**Review by SiAdcock — APPROVED** [review]
Awesome stuff

File: dev/eslint-plugin-guardian-frontend/package.json (hunk 1 above)
Comment (sndrs) [inline dev/eslint-plugin-guardian-frontend/package.json]: yeah good point. might as well in this

Comment (sndrs) [inline dev/eslint-plugin-guardian-frontend/package.json]: hmm, not sure actually. we'd want to add eslint, requireindex etc too, and then we'd need to keep them in sync with main versions. then would we install these versions or main ones? a lot of questions...will leave for now i think

**Pull request merged**